testing/test-utils/src/lib/utils/dynamic-mocks/eslint-plugin.mock.ts

```diff
@@ -2,12 +2,28 @@ import { join } from 'node:path';
 import type {
   Audit,
   CategoryRef,
+  Group,
   PluginConfig,
   PluginReport,
 } from '@code-pushup/models';
 import { ESLINT_AUDITS_MAP } from './eslint-audits.mock';
 import { echoRunnerConfigMock } from './runner-config.mock';
 
+export const ESLINT_PLUGIN_GROUP_PERFORMANCE: Group = {
+  slug: 'performance',
+  title: 'ESLint performance rules',
+  refs: [
+    {
+      slug: ESLINT_AUDITS_MAP['max-lines-per-function'].slug,
+      weight: 10,
+    },
+    {
+      slug: ESLINT_AUDITS_MAP['react-jsx-key'].slug,
+      weight: 25,
+    },
```

Comment:
The weights are kinda unintuitive - `max-lines-per-function` is ~28.6% and `react-jsx-key` is ~71.4%.

Reply:
Bruh...

Reply:
Just give them both weight 1?